core/src/main/java/org/elasticsearch/action/fieldstats/FieldStats.java

```diff
@@ -345,7 +346,7 @@ public void append(FieldStats stats) {
         }
 
         @Override
-        protected java.lang.Double valueOf(String value) {
+        protected java.lang.Double valueOf(String value, String optionalFormat) {
```

Comment:
this feels weird. We have a parameter and not use it. Can we add java docs and throw and operation not supported if someone used it? (this goes for all of those)